emperor/support_files/js/view.js

```diff
@@ -337,6 +372,125 @@ DecompositionView.prototype._fastInit = function() {
   this.markers.push(cloud);
 };
 
+/**
+ * Parallel plots closely mirroring the shader enabled _fastInit calls
+ */
+DecompositionView.prototype._fastInitParallelPlot = function()
+{
+  var positions, colors, scales, opacities, visibilities, geometry, cloud;
+
+  // We're really just drawing a bunch of line strips...
+  // highly doubt shaders are necessary for this...
+  var vertexShader = `
+    attribute float scale;
+    attribute vec3 color;
+    attribute float opacity;
+    attribute float visible;
+    
+    varying vec3 vColor;
+    varying float vOpacity;
+    varying float vVisible;
+    
+    void main() {
+      vColor = color;
+      vOpacity = opacity;
+      vVisible = visible;
+    
+      gl_Position = projectionMatrix * modelViewMatrix * vec4(position, 1.0);
+      gl_PointSize = 2.0;
+    }
+    `;
+
+  var fragmentShader = `
+    precision mediump float;
+    varying vec3 vColor;
+    varying float vOpacity;
+    varying float vVisible;
+    
+    void main() {
+      if (vVisible <= 0.0)
+        discard;
+      gl_FragColor = vec4(vColor, vOpacity);
+    }
+  `;
+
+  var allDimensions = _.range(this.decomp.dimensions);
+
+  // We'll build the line strips as GL_LINES for simplicity, at least for now,
+  // by doubling up vertex positions at each of the intermediate axes.
+  var numPoints = (allDimensions.length * 2 - 2) * (this.decomp.length);
+
+  positions = new Float32Array(numPoints * 3);
+  colors = new Float32Array(numPoints * 3);
+  scales = new Float32Array(numPoints);
```

Comment:
Can you remove this line.

Reply:
422?  I guess so?  I'm starting to wonder if your editor is showing something that I'm not seeing though.

Reply:
Not sure. Here I was referring to the "scales" attribute given that we can't really change the size of the lines, right?

Reply:
Ahh, the scale lines, done.